.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  CI:
+    name: ${{ matrix.php }}
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - php: '7.4'
+          - php: '8.0'
+          - php: '8.1'
+          - php: '8.2'
+
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          coverage: none
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install vips
+        run: sudo apt-get install -y libvips
```

Note to myself: try to fix https://github.com/libvips/libvips/discussions/2812, to avoid downloading nip2 + dependencies on Debian and derivatives. If this cannot be fixed, we may consider doing this instead:
```suggestion
        run: sudo apt install -y libvips --no-install-recommends
```

---

`--no-install-recommends` is probably the best fix, in the short term anyway.

---

And also changed. Tests still run through